### ISSUE TITLE
TypeError: Cannot read property 'find' of undefined at PurgeCSS.getFileExtractor

### ISSUE THREAD

**Issue by marekmiotelka** [FullHuman/purgecss#259]
purgecss-webpack-plugin: 2.0.3

After update from 1.6.0 to 2.0.3 Webpack Plugin stopped working with partials/string interpolation.

My code (worked in 1.6.0): 
```javascript
const purgeCSS = new PurgecssPlugin({
  paths: glob.sync(path.join(config.root, config.paths.src, '/**/*'), { nodir: true }),
});
```

Error: 

```javascript
Compilation pass 1
nameLookup partials sample-component partial
Attempting to resolve  helpers : []
Attempting to resolve  unclearStuff : []
Attempting to resolve  partials : [ '$sample-component' ]
Attempting to resolve partial /Users/marekmiotelka/Documents/Projects/StarterProjects/starter-template_connected/src/sample-component.html
request=./sample-component.html
Attempting to resolve partial /Users/marekmiotelka/Documents/Projects/StarterProjects/starter-template_connected/src/components/sample-component/sample-component.html
request=./sample-component.html
Resolved partial /Users/marekmiotelka/Documents/Projects/StarterProjects/starter-template_connected/src/components/sample-component/sample-component.html

Compilation pass 2
nameLookup partials sample-component partial
Attempting to resolve  helpers : []
Attempting to resolve  unclearStuff : []
Attempting to resolve  partials : [ '$sample-component' ]

Compilation pass 1
nameLookup depth0 title context
nameLookup helpers title helper
nameLookup helpers title helper
Attempting to resolve  helpers : [ '$title' ]
Attempting to resolve helper /Users/marekmiotelka/Documents/Projects/StarterProjects/starter-template_connected/src/components/sample-component/title
request=./title
Attempting to resolve helper /Users/marekmiotelka/Documents/Projects/StarterProjects/starter-template_connected/src/components/sample-component/title
request=./title
Failed to resolve helper /Users/marekmiotelka/Documents/Projects/StarterProjects/starter-template_connected/src/components/sample-component/title
Attempting to resolve  unclearStuff : [ '$title' ]
Attempting to resolve unclearStuff /Users/marekmiotelka/Documents/Projects/StarterProjects/starter-template_connected/src/components/sample-component/title
request=./title
Attempting to resolve unclearStuff /Users/marekmiotelka/Documents/Projects/StarterProjects/starter-template_connected/src/components/sample-component/title
request=./title
Failed to resolve unclearStuff /Users/marekmiotelka/Documents/Projects/StarterProjects/starter-template_connected/src/components/sample-component/title
Attempting to resolve  partials : []
TypeError: Cannot read property 'find' of undefined
    at PurgeCSS.getFileExtractor (/Users/marekmiotelka/Documents/Projects/StarterProjects/starter-template_connected/node_modules/purgecss/lib/purgecss.js:1:5487)
    at PurgeCSS.extractSelectorsFromFiles (/Users/marekmiotelka/Documents/Projects/StarterProjects/starter-template_connected/node_modules/purgecss/lib/purgecss.js:1:5920)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! starter-template@0.0.1 build:dist: `cross-env NODE_ENV=production webpack --config ./config/webpack.config.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the starter-template@0.0.1 build:dist script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

```

Looks like he's looking for partials in wrong directories (i use Handlebars with string interpolation).

My main HTML file contains such an import:

src/index.html
```handlebars
    {{> sample-component title='Test component'}}
```

and the partial:
src/components/sample-partial/smaple-partial.html
```handlebars
<div class="sample-component">
  <h2 class="sample-component__title">{{title}}</h2>
</div>
```

Before update everything worked fine.

**Comment by Ffloriel** [comment]
Would you have a small repository to reproduce this issue?

**Comment by Ffloriel** [comment]
The issue should be fixed in the version 2.0.4.
